src/util/util_string.c

```diff
@@ -0,0 +1,50 @@
+// Copyright 2019 Shift Cryptosecurity AG
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "util_string.h"
+
+#include <string.h>
+
+bool util_string_all_ascii_bytes(const uint8_t* bytes, size_t bytes_len)
+{
+    for (size_t i = 0; i < bytes_len; ++i) {
+        if (bytes[i] < 32 || bytes[i] > 126) {
+            return false;
+        }
+    }
+    return true;
+}
+
+bool util_string_all_ascii(const char* str)
+{
+    return util_string_all_ascii_bytes((const uint8_t*)str, strlen(str));
+}
+
+bool util_string_validate_name(const char* name, size_t max_len)
+{
```

Comment:
What do you think about an `Abort(...)` in case `name==NULL`?

Reply:
:+1: done